main/src/main/scala/sbt/internal/SettingGraph.scala

```diff
@@ -60,8 +63,7 @@ object Graph {
   // [info]   | +-baz
   // [info]   |
   // [info]   +-quux
-  def toAscii[A](top: A, children: A => Seq[A], display: A => String): String = {
-    val defaultWidth = 40
+  def toAscii[A](top: A, children: A => Seq[A], display: A => String, defaultWidth: Int): String = {
     // TODO: Fix JLine
     val maxColumn = math.max( /*JLine.usingTerminal(_.getWidth)*/ 0, defaultWidth) - 8
```

Comment:
This is not your change, but we should fix this one too from c981abd295fbb8a93d712ccefc46027772f7f18a /cc @Duhemm